metadata-ingestion/src/datahub/configuration/source_common.py

```diff
@@ -53,13 +49,13 @@ def env_must_be_one_of(cls, v: str) -> str:
         return v.upper()
 
 
-class DatasetSourceConfigBase(PlatformSourceConfigBase, EnvBasedSourceConfigBase):
+class DatasetSourceConfigMixin(PlatformInstanceConfigMixin, EnvConfigMixin):
```

Comment:
Are these all just renames to use the MixIn nomenclature?

Reply:
renames + removing the mixins in some cases